mesonbuild/dependencies/base.py

```diff
@@ -24,7 +24,7 @@
 from ..compilers import clib_langs
 from ..mesonlib import LibType, MachineChoice, MesonException, HoldableObject
 from ..mesonlib import version_compare_many
-from ..interpreterbase import FeatureDeprecated
+#from ..interpreterbase import FeatureDeprecated, FeatureNew
```

Comment:
this looks like something half finished?

Reply:
It's part of the FIXME down below, we want to do Feature checks in this file but we cannot.